### ISSUE TITLE
Some dungeon quests not shown in map

### ISSUE THREAD

**Issue by kenhwong** [IrcDirk/Carbonite-Classic#7]
Some dungeon quests not shown in map(my list have 3 items: 1951-Rituals of Power, 1956-Power in Uldaman, 2846-Tiara of the Deep ), and alert 'This quest is not in the database'.

**Comment by IrcDirk** [comment]
Will check that later.

**Comment by kenhwong** [comment]
Normal 40+ quests shown and waypoint works fine. but these in dungeon still alert quest target not in the database.

**Comment by IrcDirk** [comment]
Didnt test instances and BG yet.

**Comment by IrcDirk** [comment]
I think problem is with objectives that are not in database becouse they are not in WoWHead database. All quest data is scrapped from WoWHead.

**Comment by axlekitty** [comment]
there is a wowhead addon if you guys would like to gather and contribute to the quests not in the database. its how we did it back in the day.

**Comment by wiesshund** [comment]
Is the instance maps part of this issue (not the quest indicators part) still being worked on?
Or is there anything we could do to help?

**Comment by IrcDirk** [comment]
No, im not working on this. I dont have that many time... just fixing recent issues. All users are welcome to help. Any one can make pull requests from his github fork of Carbonite :)

**Comment by IrcDirk** [comment]
First try to fix this => https://github.com/IrcDirk/Carbonite-Classic/commit/b24806e5850c9edcf6eb0a634a3a2574f1706e4a

WOTLK instances should show the map (only Northen instances have maps in game), rest will maximize Minimap.
